app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -12,18 +12,58 @@
         <%= t '.section_1.heading' %>
       </dt>
 
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_1.proceeding'), answer: @proceeding.meaning } %>
+      <%= check_provider_answer_link(
+            name: 'proceeding_type',
+            url: providers_legal_aid_application_proceedings_type_path,
+            question: t('.section_1.proceeding'),
+            answer: @proceeding.meaning
+          ) %>
 
       <dt class="govuk-heading-m govuk-!-padding-top-9">
         <%= t '.section_2.heading' %>
       </dt>
 
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_2.first_name'), answer: @applicant.first_name } %>
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_2.last_name'), answer: @applicant.last_name } %>
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_2.dob'), answer: @applicant.date_of_birth.to_s } %>
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_2.nino'), answer: @applicant.national_insurance_number } %>
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_2.address'), answer: address_with_line_breaks(@address) } %>
-      <%= render partial: 'check_your_answers_item', locals: { question: t('.section_2.email'), answer: @applicant.email } %>
+      <%= check_provider_answer_link(
+            name: 'first_name',
+            url: providers_legal_aid_application_applicant_path,
+            question: t('.section_2.first_name'),
+            answer: @applicant.first_name
+          ) %>
+
+      <%= check_provider_answer_link(
+            name: 'last_name',
+            url: providers_legal_aid_application_applicant_path,
+            question: t('.section_2.last_name'),
+            answer: @applicant.last_name
+          ) %>
+
+      <%= check_provider_answer_link(
+            name: 'date_of_birth',
+            url: providers_legal_aid_application_applicant_path,
+            question: t('.section_2.dob'),
+            answer: @applicant.date_of_birth
+          ) %>
+
+      <%= check_provider_answer_link(
+            name: 'national_insurance_number',
+            url: providers_legal_aid_application_applicant_path,
+            question: t('.section_2.nino'),
+            answer: @applicant.national_insurance_number
+          ) %>
+
+      <%= check_provider_answer_link(
+            name: 'address',
+            url: providers_legal_aid_application_address_lookup_path,
+            question: t('.section_2.address'),
+            answer: address_with_line_breaks(@address)
+          ) %>
+
+      <%= check_provider_answer_link(
+            name: 'email',
+            url: providers_legal_aid_application_email_path,
```

Comment:
This needs to go to the client details page now (`providers_legal_aid_application_applicant_path`).
Would you mind changing it? :smiley:

Reply:
Yes - I noticed that. If your PR is merged in, I'll do it as part of the rebase work.